### PR TITLE
Landing page

### LINKED ISSUE #1: Setup Project
As a developer, 
I want to know 
in which framework I will work
and create a repository in this framework
- [x] Decision about framework
- [x] Implement decision

### LINKED ISSUE #2: Setup CD (continuous deployment) Pipeline
As a developer,
I want to see my result,
so that I can test it

AC:
- [x] Connect to Vercel

### LINKED ISSUE #3: Create Repo and Agile Board
- [x] create the repo using template from pixelass
- [x] create agile board and copy previously created tickets to this board
- [x] clone repo to machine

### PR DESCRIPTION
**Pull request by Git-Hannah**
## Motivation

- Created the next.js setup from template
- setup initial landing page with placeholder hero image
- initialized emotion styled components
- initialized mui material

## Issues closed

closes #1 
closes #2 
closes #3

### CODE REVIEW

File: src/pages/copy-of-index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+			)}
+			<Button>Click me</Button>
+			<Link href="/foobar">
+				<a>Foobar</a>
+			</Link>
+		</Layout>
+	);
+};
+
+export default Page;
```
Comment (marcbendixen) [inline src/pages/copy-of-index.jsx:33]: I assume this file is only a backup!? If so, you can delete it.

File: src/pages/index.jsx [hunk 2]
```diff
@@ -1,28 +1,39 @@
...
-const Page = () => {
-	const { data, loading, error } = useGet("/api/hello");
+// image link:
+// https://images.unsplash.com/photo-1514306191717-452ec28c7814?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1169&q=80
+
+const StyledImage = styled(Image)`
+	border-radius: 100px;
+`;
 
+const A = ({ children, className }) => {
```
Comment (marcbendixen) [inline src/pages/index.jsx:13]: `A` isn't a good naming convention for a `const`. Try to find a better name.

File: src/pages/index.jsx [hunk 3]
```diff
@@ -1,28 +1,39 @@
...
+	return (
+		<div className={className}>
+			{children}
+		</div>
+	)
+}
+const B = styled(A)`
```
Comment (marcbendixen) [inline src/pages/index.jsx:20]: `B` isn't a good naming convention for a `const`. Try to find a better name.

**Review by marcbendixen — CHANGES_REQUESTED** [review]
Good job 🚀 
I added some comments.

File: src/pages/copy-of-index.jsx (hunk 1 above)
Comment (Git-Hannah) [reply src/pages/copy-of-index.jsx:33]: Yes, it is my Spickzettel! :)

File: src/pages/index.jsx (hunk 2 above)
Comment (Git-Hannah) [reply src/pages/index.jsx:13]: Thank you, that is correct, it was an exercise, and will be fixed!

File: src/pages/index.jsx (hunk 3 above)
Comment (Git-Hannah) [reply src/pages/index.jsx:20]: Thank you, that is correct, it was an exercise, and will be fixed!

**Pull request merged**